### ISSUE TITLE
Use outside meteor?

### ISSUE THREAD

**Issue by thebarty** [vazco/uniforms#409]
Hi guys,

I'd love to use this package within gatsby, but I am getting those errors, because I can not install meteor packages?
```
* meteor/check in ./~/uniforms/SimpleSchemaBridge.js
* meteor/aldeed:simple-schema in ./~/uniforms/SimpleSchemaBridge.js
```
Any ideas?

**Comment by radekmie** [comment]
Yeah, it's a known issue - every static analysis will raise a warning about these modules. Anyway, it's _only_ (I know, I hate them too) a warning.
